src/pages/ProxyVotes.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+
+import IntlMessage from "../components/IntlMessage";
+import { AppState } from "../reducers";
+import {
+  Member,
+  MemberLookupTable,
+  GENESIS_MEMBER
+} from "../reducers/membersNew";
+
+interface OwnProps {}
+interface StateProps {
+  members: Member[];
+}
+interface DispatchProps {}
+interface Props {
+  memberAndVotes: [[Member, number]];
+}
+
+const ProxyVotesElem = styled.main`
+  max-width: 90vw;
+  display: flex;
```

Comment:
P2: Why does this need to be a flex component? It's just a list, no need to make the style more complex than it needs to be.